src/ui/public/doc_table/components/table_row/cell.html

```diff
@@ -13,17 +13,15 @@
   <span class="table-cell-filter">
     <% if (filterable) { %>
       <span
-        ng-click="inlineFilter($event, '+')"
+        ng-click="inlineFilter('<%- column %>', '+')"
```

Comment:
We tried passing the column name directly like this in the original PR but there appeared to be encoding issues: https://github.com/elastic/kibana/pull/5344/files#r95694856